src/pages/guides/using/marketo/marketo-lead-activity-data-stream-setup.md

```diff
@@ -279,12 +272,6 @@ The `data` field contains the following data:
 | munchkinId              | String            | The Munchkin ID of the Marketo instance                             |
 | leadId                  | String (Number)   | The ID of the lead that the event is related to                     |
 | lead                    | Object            | (**optional**) The lead object that the event related to            |
```

Comment:
One more to get rid of, the lead object itself. The default is just leadId, the rest is feature flagged so not part of standard availability, otherwise we have like 40 other fields to try to include as well that are optional and I don't want to do that here